atoms/badge/index.css

```diff
@@ -0,0 +1,28 @@
+.tui__badge-xs {
+  padding: var(--padding-xs);
+  border-radius: var(--badge-border-radius-lg);
+}
+.tui__badge-pos-abs-tr {
+  position: absolute;
+  top: 0%;
```

Comment:
add position base/utility class if feasible

Reply:
As discussed over the call these are absolute properties and we don't need to create a separate class as it as these values are specific to the utility class. In case we need to change these values here then we may have to reconsider having more utility classes to cater to the purpose itself.